src/api/public/apidocs-new/paths/source_project_name_cmd_showlinked.yaml

```
@@ -0,0 +1,43 @@
+post:
+  summary: List projects linking to the current project.
+  description: |
+    Return a list of projects linking to the project passed as parameter. This command doesn't perform any action.
```

I think this would read better. What do you think?

```suggestion
    Return a list of projects linking to the provided project. This command doesn't perform any action.
```

---

Yes, way better. Thanks! Applied.